.travis.yml

```diff
@@ -15,5 +15,14 @@ env:
       # install timeout in minutes
       - ADB_INSTALL_TIMEOUT=8
 
+# Create and start the emulator
+before_script:
+   - echo no | android create avd --force -n test -t android-21  --abi armeabi-v7a -c 128M
```

Comment:
Apparently there are some issues with the x86 emulator:

https://github.com/travis-ci/travis-ci/issues/1419

But based on some suggestions on that thread, I could use the -PdisablePreDex to speed up the build.